viz/metrics-api/stat_summary_test.go

```diff
@@ -1400,6 +1400,50 @@ status:
 										},
 									},
 								},
+								{
+									Table: &pb.StatTable_PodGroup_{
+										PodGroup: &pb.StatTable_PodGroup{
+											Rows: []*pb.StatTable_PodGroup_Row{
+												{
+													Resource: &pb.Resource{
+														Type:      pkgK8s.Server,
+														Namespace: "emojivoto",
+													},
+													TimeWindow: "1m",
+													Stats: &pb.BasicStats{
+														SuccessCount: 123,
+														FailureCount: 0,
+														LatencyMsP50: 123,
+														LatencyMsP95: 123,
+														LatencyMsP99: 123,
+													},
+												},
+											},
+										},
+									},
+								},
+								{
+									Table: &pb.StatTable_PodGroup_{
+										PodGroup: &pb.StatTable_PodGroup{
+											Rows: []*pb.StatTable_PodGroup_Row{
+												{
+													Resource: &pb.Resource{
+														Type:      pkgK8s.ServerAuthorization,
+														Namespace: "emojivoto",
+													},
+													TimeWindow: "1m",
+													Stats: &pb.BasicStats{
+														SuccessCount: 123,
+														FailureCount: 0,
+														LatencyMsP50: 123,
+														LatencyMsP95: 123,
+														LatencyMsP99: 123,
+													},
+												},
+											},
+										},
+									},
+								},
```

Comment:
So these unit tests will get completed with srv and saz objects in the `k8sConfigs` fixtures?